go.mod

```diff
@@ -16,7 +16,7 @@ require (
 	github.com/spf13/pflag v1.0.5
 	go.uber.org/atomic v1.4.0 // indirect
 	go.uber.org/zap v1.10.0 // indirect
-	golang.org/x/crypto v0.0.0-20190911031432-227b76d455e7 // indirect
+	golang.org/x/crypto v0.0.0-20190911031432-227b76d455e7
```

Comment:
Are you using Go 1.13 locally?

Reply:
This is correct actually. Comes from `go mod tidy`. Importing crypto directly for blake2b, so the indirect ref got removed.